### PR TITLE
Simple change to add missing Javadoc to class header and public method.

### PR DESCRIPTION
**Pull request by hsaputra**
Another simple PR to add missing Javadoc to class header and public method.

### CODE REVIEW

File: twill-zookeeper/src/main/java/org/apache/twill/zookeeper/ZKClients.java [hunk 1]
```diff
@@ -51,7 +51,13 @@ public static ZKClient retryOnFailure(ZKClient client, RetryStrategy retryStrate
     return new FailureRetryZKClient(client, retryStrategy);
   }
 
-
+  /**
+   * Creates a {@link ZKClient} that will add prefix namespace for every paths.
+   *
+   * @param zkClient The {@link ZKClient} for operations delegation.
+   * @param namespace The prefix namespace to be appended to paths.
+   * @return
```
Comment (poornachandra) [inline twill-zookeeper/src/main/java/org/apache/twill/zookeeper/ZKClients.java]: Empty return tag

File: twill-zookeeper/src/main/java/org/apache/twill/zookeeper/ZKClients.java [hunk 2]
```diff
@@ -51,7 +51,13 @@ public static ZKClient retryOnFailure(ZKClient client, RetryStrategy retryStrate
     return new FailureRetryZKClient(client, retryStrategy);
   }
 
-
+  /**
+   * Creates a {@link ZKClient} that will add prefix namespace for every paths.
+   *
+   * @param zkClient The {@link ZKClient} for operations delegation.
+   * @param namespace The prefix namespace to be appended to paths.
```
Comment (poornachandra) [inline twill-zookeeper/src/main/java/org/apache/twill/zookeeper/ZKClients.java]: Isn't it better to use prepend instead of append here?

**Comment by poornachandra** [comment]
Two minor comments, otherwise LGTM.

**Comment by hsaputra** [comment]
Thanks @poornachandra , just updated PR based on review.

**Comment by chtyim** [comment]
Please rebase to one commit.

**Comment by hsaputra** [comment]
Thanks, just rebased into one commit.

**Pull request closed without merging**